backend/drive/drive.go

```diff
@@ -3570,6 +3617,19 @@ Use the --interactive/-i or --dry-run flag to see what would be copied before co
 }, {
 	Name:  "importformats",
 	Short: "Dump the import formats for debug purposes",
+}, {
+	Name:  "query",
+	Short: "List files using Google Drive query language",
+	Long: `This command lists files based on a query
+
+Usage:
+
+    rclone backend query drive: query
+    
+The query syntax is documented at [Google Drive Search query terms and 
+operators](https://developers.google.com/drive/api/guides/ref-search-terms).
+
+Results are tab-separated id, name pairs.`,
```

Comment:
My instincts are to have a more fully featured output format than TSV like JSON as it is difficult to predict what people want from this sort of command in advance and adding things to JSON is much easier without breaking backwards compatibility.

I'd probably just return a `[]*drive.File` from `query` and then return that from the backend command which will JSON-ify it.

What do you think?

We could always have an option `-o format=tsv` ?

Reply:
I didn't realise it would produce JSON for free! I've removed the TSV formatting and will adjust our calling code or pipe the output through jq as needed.